ngraph/test/backend/region_yolo.in.cpp

```diff
@@ -70,3 +70,33 @@ NGRAPH_TEST(${BACKEND_NAME}, region_yolo_v3_mxnet)
         shape, TEST_FILES, "region_out_yolov3_mxnet.data");
     test_case.run_with_tolerance_as_fp(1.0e-4f);
 }
+
+NGRAPH_TEST(${BACKEND_NAME}, region_yolo_v3_mxnet_2)
+{
+    const size_t num = 3;
+    const size_t coords = 4;
+    const size_t classes = 20;
+    const size_t batch = 1;
+    const size_t channels = 303;
+    const size_t width = 28;
+    const size_t height = 28;
+    const std::vector<int64_t> mask{0, 1, 2};
+    const int axis = 1;
+    const int end_axis = 3;
+
+    Shape input_shape{batch, channels, height, width};
+    Shape output_shape{batch, (classes + coords + 1) * mask.size(), height, width};
+
+    const auto A = make_shared<op::Parameter>(element::f32, input_shape);
+    const auto R = make_shared<op::v0::RegionYolo>(A, coords, classes, num, false, mask, axis, end_axis);
+    const auto f = make_shared<Function>(R, ParameterVector{A});
+
+    EXPECT_EQ(R->get_output_shape(0), output_shape);
+
+    auto test_case = test::TestCase<TestEngine>(f);
+
+    test_case.add_input_from_file<float>(input_shape, TEST_FILES, "region_in_yolov3_mxnet_2.data");
```

Comment:
Do not add more binary files with test data! Let's generate random data or prepare a small test hardcoded as a plain text in the test itself.

Reply:
Reimplemented